src/main/java/de/tum/in/www1/artemis/service/QuizExerciseService.java

```diff
@@ -353,6 +357,11 @@ public void sendQuizExerciseToSubscribedClients(QuizExercise quizExercise, Strin
             byte[] payload = objectMapper.writerWithView(view).writeValueAsBytes(quizExercise);
             // For each change we send the same message. The client needs to decide how to handle the date based on the quiz status
             if (quizExercise.isVisibleToStudents() && quizExercise.hasCourse()) {
+                // Create a group notification if actions is 'start-now'.
+                if ("start-now".equals(quizChange)) {
```

Comment:
I would suggest moving start-now into a shared constant wherever it is used. Can quizChange be an Enum instead of just a string?